app/src/main/java/swati4star/createpdf/util/PopulateList.java

```diff
@@ -69,7 +69,9 @@ private void populateListView() {
             mHandler.post(mEmptyStateChangeListener::showNoPermissionsView);
         else if (pdfFiles.size() == 0) {
             mHandler.post(mEmptyStateChangeListener::setEmptyStateVisible);
+            mHandler.post(() -> mAdapter.setData(null));
```

Comment:
This is also going to affect when there are no PDFs to show initially also

Reply:
I tried this, the behavior when there are no PDFs initially remains the same, the message shows up, same thing if you delete all the pdfs